.travis.yml

```diff
@@ -29,6 +30,14 @@ install:
 before_script:
   - docker version
   - docker-compose version
+  # checkout the ICDS code from it's own repo
```

Comment:
Is this expected to be a permanent part of test runs?

Reply:
For the time being yes. We don't have a way to run the ICDS tests outside of HQ at present. Also there's enough dependency between the two code bases that we want to make sure they continue to work together. 

I would hope we can change this in the future but for now this is necessary.